docs/virtual_machines/.pages

```diff
@@ -10,6 +10,7 @@ nav:
   - numa.md
   - disks_and_volumes.md
   - interfaces_and_networks.md
+  - Network Binding Plugins: net_binding_plugins
```

Comment:
Please move it to the end of the menu. It is now breaking the otherwise flat menu in half:

![image](https://github.com/kubevirt/user-guide/assets/3493830/07f6df60-0b14-4e1c-a09d-116b25e53aac)

Reply:
Done

Reply:
![binding](https://github.com/kubevirt/user-guide/assets/17478564/299326b2-0b69-47a3-8944-dba79c3f1f8a)
I think that passt and slirp should be in the tree.